DependencyInjection/Configuration.php

```diff
@@ -107,7 +107,11 @@ public function getConfigTreeBuilder()
                                 return $v;
                             })
                         ->end()
-                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))->end()
+                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))
+                    ->end()
+                    ->scalarNode('service_naming_strategy')
```

Comment:
Do you want to override this from another bundle, or from your application? It seems like this would not make sense to override from a re-usable bundle, but you would do that for your application only.